core/org.openjdk.jmc.agent/src/main/java/org/openjdk/jmc/agent/impl/DefaultTransformRegistry.java

```diff
@@ -150,11 +152,12 @@ private static TransformDescriptor parseTransformData(
 		Map<String, String> values = new HashMap<>();
 		List<Parameter> parameters = new LinkedList<>();
 		Method method = null;
+		ReturnValue[] returnValue = new ReturnValue[1];
```

Comment:
Having an array of 1 seems unnecessary here. Wouldn't just having a reference to the ReturnValue be cleaner?

Reply:
I made an array as a wrapper so I do not lose the reference of the ReturnValue object created inside the callee function.  The reference to the newly created object gets lost if I pass in a ReturnValue reference that initially is set to null.

I could create an empty ReturnValue object and then pass it into the parseMethod function, where the object's fields can be set, instead of allocating an array of size 1 ?